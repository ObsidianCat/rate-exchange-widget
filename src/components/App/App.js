import React, { Component } from 'react';
import 'antd/dist/antd.css';
import './App.css';
import Exchanger from "../Exchanger/Exchanger";
import store from "../../store";
import { Provider } from "react-redux";

class App extends Component {
  render() {
    return (
      <div className="App">
        <Provider store={store}>
          <Exchanger/>
        </Provider>
      </div>
    );
  }
}

export default App;
