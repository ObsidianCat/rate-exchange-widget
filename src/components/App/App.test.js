import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

let wrapper;

beforeEach(()=>{
  wrapper =  shallow(<App />);
})

afterEach(()=>{
  wrapper.unmount()
})

it('renders as snapshot', () => {
  expect(wrapper).toMatchSnapshot();

});
