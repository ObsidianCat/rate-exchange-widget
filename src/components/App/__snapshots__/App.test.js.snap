// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as snapshot 1`] = `
<div
  className="App"
>
  <Provider
    store={
      Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      }
    }
  >
    <Connect(ExchangerComponent) />
  </Provider>
</div>
`;
